opentech/apply/funds/templates/funds/includes/status_bar.html

```diff
@@ -1,27 +1,26 @@
 <div class="status-bar {{ class }}">
-    {% for phase in phases %}
-        <div class="status-bar__item
-                    {% if phase == status %}
-                        status-bar__item--is-current
-                    {% elif phase < status %}
-                        status-bar__item--is-complete
-                    {% endif %}">
-            <span class="status-bar__tooltip"
-                {% if phase == status %}
-                    {# We want to display the status explicitly in case phase is a MultiStep (displays "Outcome" for name) #}
```

Comment:
I am still not convinced by showing the first value from multiple step phases.
When something is Under discussion, but needs more info it makes sense to show "Under discussion: needs more info", and Outcome reads much better than "Accepted", even if we are not there.